src/store/store.js

```diff
@@ -12,6 +12,7 @@ export const topology = writable({
   "node_containers": [],
 })
 export const containers = writable([])
+export const visibleContainer = writable("wtf")
```

Comment:
tu

Reply:
```suggestion
export const visibleContainer = writable("")
```
maybe set variable to empty string, the value was set this way for debugging reasons